homeassistant/components/zwave_js/services.py

```diff
@@ -94,15 +94,13 @@ def async_register(self) -> None:
                 {
                     vol.Optional(ATTR_DEVICE_ID): vol.All(cv.ensure_list, [cv.string]),
                     vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
-                    vol.Required(const.ATTR_CONFIG_PARAMETER): vol.Any(
-                        vol.Coerce(int), cv.string
-                    ),
+                    vol.Required(const.ATTR_CONFIG_PARAMETER): vol.Coerce(int),
```

Comment:
Should we mark this change as a breaking change?

Reply:
It's not a breaking change because the service never supported a string as an input - because of the logic here, we always expected an int (`property_` is always an int for config parameters) - https://github.com/home-assistant-libs/zwave-js-server-python/blob/a93ede6341ac0972aba3335fbe962c7d9c790688/zwave_js_server/util/node.py#L88-L92

So the main change here is instead of raising an exception from the library, we raise a service schema exception